Added full Spanish translation (by eduardo_a2j)

xoascf opened the pull request:
Based on US ROM v1.0
http://dorando.emuverse.com/projects/eduardo_a2j/zelda-ocarina-of-time.html

xoascf committed:
Added full Spanish translation (by eduardo_a2j)

Based on US ROM v1.0
http://dorando.emuverse.com/projects/eduardo_a2j/zelda-ocarina-of-time.html

xoascf committed:
Update message_data_es-SV.h

RyzenDew commented:
This won’t be merged until you give proof the maker of this gave you expression permission.

xoascf committed:
Fix size bug, use correct sound for Deku Scrubs

One error in the original translation was that the Deku Scrubs sounded like Wolfos

xoascf commented:
> This won’t be merged until you give proof the maker of this gave you expression permission.

![imagen](https://user-images.githubusercontent.com/28744392/212211008-4c671cd4-9c5c-4181-9da1-acb8498803dc.png)

_Yes, of course, as long as you leave the translation credits, you can use it without any problem._

xoascf commented:
@blawar, should we be able to use localized `message_data_staff.h` to include the translation credits?

xoascf committed:
Update message_data_es-SV.h

Swap TEXTBOX_POS_MIDDLE with TEXTBOX_POS_BOTTOM
